bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/CloneUtils.java

```diff
@@ -1,7 +1,7 @@
 /*
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
```

Comment:
Why we are changing this to `2023`? The file is being existed since `2019`

Reply:
Here I have moved the `CloneUtils` class to another package by deleting the previous file and creating a new file with the same content. In this case do I need to keep the previous year?

Reply:
Changed the year back to 2019.